test/linters_test.go

```diff
@@ -49,6 +49,9 @@ func testSourcesFromDir(t *testing.T, dir string) {
 
 	for _, s := range sources {
 		s := s
+		if s != "testdata/nolintlint.go" {
```

Comment:
Is this necessary? Looks left over from debugging/testing.

Reply:
No, have removed.